app/views/alchemy/base/500.html.erb

```diff
@@ -12,15 +12,29 @@
   ], search: false
 ) %>
 
+<% content_for(:alchemy_body_class) { 'error' } %>
+
```

Comment:
The `error.message` class is used all over the place. And we don't want the height and overflow scroll statements on these elements.